src/components/subscriptions/licenses/LicenseManagementTable/LicenseManagementTableBulkActions.jsx

```diff
@@ -195,7 +195,7 @@ const LicenseManagementTableBulkActions = ({
         <ModalPopup positionRef={target} isOpen={isOpen} onClose={close}>
           <div>
             <Button
-              variant="outline-danger"
+              variant="danger"
```

Comment:
Christ this looks good, the only concern is that this makes the styling on the buttons inconsistent.

Additionally, it actually doesn't address the actual issue that the `ModalPopup` background is transparent. IIRC @adamstankiewicz said the reason it might be transparent is that the development environments weren't properly loading the edx theme. In production, it works as intended.

I also remember this being added as a grooming note on the ticket but maybe I'm remembering incorrectly or it wasn't saved.

Reply:
it does appear it renders correctly on stage. I don't remember the conversation, and it's not in the ticket, so good call out 😅 

i can look into the theme bit